base_layer/core/src/base_node/chain_metadata_service/service.rs

```diff
@@ -85,19 +88,27 @@ impl ChainMetadataService {
             tokio::select! {
                 Ok(block_event) = block_event_stream.recv() => {
                     log_if_error!(
-                        level: debug,
+                        level: warn,
                         target: LOG_TARGET,
                         "Failed to handle block event because '{}'",
                         self.handle_block_event(&block_event).await
                     );
                 },
 
                 Ok(event) = liveness_event_stream.recv() => {
-                    log_if_error!(
-                        target: LOG_TARGET,
-                        "Failed to handle liveness event because '{}'",
-                        self.handle_liveness_event(&event).await
-                    );
+                    match
+                        self.handle_liveness_event(&event).await {
+                        Ok(_) => {}
+                        Err(e) => {
+                           error!( target: LOG_TARGET, "Failed to handle liveness event because '{}'", e);
+                           if let ChainMetadataSyncError::ReceivedInvalidChainMetadata(node_id,reason) = e {
```

Comment:
nit, i'm surprised rustfmt didn't fix this.
```suggestion
                           if let ChainMetadataSyncError::ReceivedInvalidChainMetadata(node_id, reason) = e {
```

Reply:
rustfmt can't seem to format anything in a `tokio::select` I've tried to make it neat manually, but it's the wild west of fmtland

Reply:
[This issue](https://github.com/tokio-rs/tokio/issues/5441) on it was closed.